src/odemis/gui/comp/legend.py

```diff
@@ -365,3 +367,93 @@ def on_paint(self, _):
             self._max_tick_width = max_width
             self.SetMinSize((self._max_tick_width + 14, -1))
             self.Parent.GetSizer().Layout()
+
+
+class RadioLegend(wx.Panel):
+    """
+    This class describes a legend containing radio buttons,
+    where each button represents a possible selection of a VA.
+    """
+
+    def __init__(self, parent, wid=-1, pos=(0, 0), size=wx.DefaultSize, style=wx.NO_BORDER):
+        wx.Panel.__init__(self, parent, wid, pos, size, style)
+
+        self.SetBackgroundColour(parent.GetBackgroundColour())
+        self.SetForegroundColour(parent.GetForegroundColour())
+
+        # get colors: if ar-viewport is selected, to avoid colors for bg and fg for txt will be taken from selection
```

Comment:
I'm not sure I understand the comment. You mean that if you directly use `parent.GetBackgroundColour()` after, and the viewport is selected, then the value returned will be the "active/blue", which is not what we want?

Reply:
Please move this code to be first (just after the `super()`), and use `bg_color`/`fg_color` everywhere else in the class.